update invite user (#19366)
piwik/piwik: CHANGELOG.md

The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)** lets you see more details about any Matomo release, such as the list of new guides and FAQs, security fixes, and links to all closed issues. 

## Matomo 4.11.0

### Breaking Changes

> * The user management UI no longer allows to directly create new users (with a password). Instead it is possible to send an invitation to a user. Directly creating new user is still possible using the API though.
> 

Reviewer: ```suggestion
* The user management UI no longer allows direct creation of a new user (with a password). Instead an invitation can be sent via email. Directly creating a new user is still possible using the API.

```

#### New PHP events

> * Added new event `UsersManager.inviteUser.end`, which is triggered after a new users has been invited

Reviewer: ```suggestion
* Added new event `UsersManager.inviteUser.end`, which is triggered after a new user has been invited
```
